test/lib/utils/addresses.ts

```diff
@@ -25,14 +25,10 @@ export const TOKENS: Record<number, Record<string, TestToken>> = {
             decimals: 6,
             slot: 2,
         },
-        bb_a_WETH: {
+        vETH: {
```

Comment:
I believe this is already on main as well - maybe merge main into your branch to clean this up?